deploy/base/organization-catalog-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    fdk.service: organization-catalog
+  name: organization-catalog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      fdk.service: organization-catalog
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        fdk.service: organization-catalog
+    spec:
+      containers:
+        - name: organization-catalog
+          image: organization-catalog
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          resources:
+            requests:
+              memory: "350Mi"
```

Comment:
@valosnah burde kanskje legge på mer her?

Reply:
@torleifg siden det med dette blir lagt til en hard grense for minnebruk av podden så kan du i tillegg oppgi max java heap i tillegg `-Xmx`, bare legg det til i dockerfile